websites/M/Manga Plus/presence.ts

```diff
@@ -2,26 +2,41 @@ const presence = new Presence({
 		clientId: "923893773048619008"
 	}),
 	browsingTimestamp = Math.floor(Date.now() / 1000);
+let cacheMangaURL: string, cacheMangaChapter: string;
 
-presence.on("UpdateData", () => {
+presence.on("UpdateData", async () => {
 	const presenceData: PresenceData = {
+			details: "Browsing",
 			largeImageKey: "logo",
 			startTimestamp: browsingTimestamp
 		},
-		path = document.location.pathname;
-	if (path.startsWith("/updates"))
+		{ pathname } = document.location,
+		MangaURL = document.querySelector<HTMLAnchorElement>(
+			"#app div.Navigation-module_detailContainer_1aDk8 > a"
+		),
+		MangaChapter = document.querySelector<HTMLParagraphElement>(
```

Comment:
*camelCase* is preferred

Reply:
I thought consistency with the other `cache...` variables is preferred but ayt